src/engine/defaults.ts

```diff
@@ -1,5 +1,6 @@
 export const defaults = {
   dir: 'dist',
+  dest: '.',
```

Comment:
I think dist is better name

Reply:
`dest` is the option that needs to be passed to `gh-pages`: https://www.npmjs.com/package/gh-pages#user-content-optionsdest